core/modules/system/system.theme.inc

```diff
@@ -178,6 +178,9 @@ function theme_system_modules_fieldset($variables) {
     if ($required_by) {
       $requirements .= '<div class="admin-requirements">' . t('Required by:') . ' ' . implode(', ', $module['#required_by']) . '</div>';
     }
+    if (!$module['#project']) {
```

Comment:
We want these links for contrib modules, and only contrib would have a `project` key, so this should be 
```
    if (isset($module['#project'])) {
```

Reply:
Done!